### DISCUSSION TITLE
[2021.10.19] 랜선 자르기

### DISCUSSION THREAD

**Discussion by leechang4** [pyunji/Hyundai-ITnE-Team_QuadCore#144] [Algorithm]
[랜선 자르기](https://www.acmicpc.net/problem/1654)

**Comment by hyk339** [comment]
[랜선자르기 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/baekjoon/BOJ_1654.java)

**Comment by pyunji** [comment]
[랜선자르기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/BOJ/BOJ_1654.java)

**Comment by chaeyeon4u** [comment]
[랜선자르기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/BOJ_1654(2).java)

**Comment by leechang4** [comment]
[랜선자르기 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/BOJ/BOJ%201654.java)
